css/index.css

```diff
@@ -1,11 +1,36 @@
-.row {
-    background-color:  rgba(251, 147, 0, .5);
-    }
+.container{
+    height: 400px;
+    width:600;
+    display: flex;
+    justify-content: space-evenly;
+    background-color: backdrop #F5E6CA
```

Comment:
This color isn't part of the palette we chose - also can remove the backdrop keyword, not sure what this does

Reply:
It is the light tan color on the paint palette chosen